### ISSUE TITLE
pip package incompatible with numpy 1.25

### ISSUE THREAD

**Issue by dataspider** [remix/partridge#81]
* partridge version: 1.1.1
* Python version: 3.10
* Operating System: macOS Ventura 13.4

### Description

I was trying to use partridge, first from peartree, then directly, in a mamba/conda environment with numpy 1.25. 
I ran into the same issue documented by another peartree user [here](https://github.com/kuanb/peartree/issues/178): 
The code available on pip still uses np.unicode in gtfs.py, which was deprecated in numpy 1.20 and removed in numpy 1.24.

I was able to fix this for myself by replacing the pip version of partridge with the repository version _because the repository version appears to have already fixed this bug_. 
So you could make many people's life a tiny bit easier by releasing the bug-fixed version on pip :)

### What I Did

```
file = "my_awesome_gtfs_data.zip"
ptg.read_busiest_date(file)
Traceback: See https://github.com/kuanb/peartree/issues/178 (my trace was the same in all important respects)
```

**Comment by tweska** [comment]
I also encountered this issue today. Would be nice if someone can update the pypi package.

Meanwhile, for anyone who wants to install the latest version from GitHub, you can use this command:
```
pip install git+https://github.com/remix/partridge.git
```

**Comment by kuanb** [comment]
See issue notes here: https://github.com/kuanb/peartree/issues/178

TLDR is that you can get around this by installing `numpy==1.23.5` or earlier.
